src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -91,7 +91,7 @@ public without sharing class UTIL_CustomSettingsFacade {
      * Private methods that will configure the default org-level custom settings.
      **/
     private static void configSettings(Hierarchy_Settings__c hs) {
-		hs.Account_Processor__c = 'Administrative';
+		hs.Account_Processor__c = UTIL_Describe.getRecTypesMapByDevName('Account').get('Administrative');
```

Comment:
It shouldn't... The get method will just return null if the record type doesn't exist. It would only blow up if UTIL_Describe.getRecTypesMapByDevName('Account') returns null, which shouldn't happen, afaik.